vpr/src/pack/re_cluster.cpp

```diff
@@ -157,6 +165,8 @@ bool swap_two_molecules(t_pack_molecule* molecule_1,
                         bool during_packing,
                         int verbosity,
                         t_clustering_data& clustering_data) {
+    auto& cluster_ctx = g_vpr_ctx.mutable_clustering();
```

Comment:
I put this on Amin's MR, but why is this declared so far away from where it is used? Is this a style thing?